ipn/ipnlocal/profiles.go

```diff
@@ -394,15 +396,29 @@ func (pm *profileManager) writeKnownProfiles() error {
 func (pm *profileManager) NewProfile() {
 	metricNewProfile.Add(1)
 
-	pm.prefs = emptyPrefs
+	pm.prefs = defaultPrefs
 	pm.isNewProfile = true
 	pm.currentProfile = &ipn.LoginProfile{}
 }
 
-// emptyPrefs is the default prefs for a new profile.
-var emptyPrefs = func() ipn.PrefsView {
+// defaultPrefs is the default prefs for a new profile.
+var defaultPrefs = func() ipn.PrefsView {
 	prefs := ipn.NewPrefs()
 	prefs.WantRunning = false
+
+	prefs.ControlURL = winutil.GetPolicyString("LoginURL", "")
```

Comment:
It seems unfortunate to have knowledge/application of policies split between the OSS and corp repos. There's some very similar code for this at https://github.com/tailscale/corp/blob/33299d68f14826899283270da8455013d56b8014/win/ui/ui.go#L387-L406 (refactored from https://github.com/tailscale/corp/commit/35735329841815a892220aec59d9c91f97c51fc4).

One alternate implementation would be to not add Windows-specific logic here, and instead have a `defaultPrefsMutatorFn` hook (a `func(prefs *ipn.Prefs)`) that can be set by the clients that use it. The Windows client could then set it with a function that does the policy (and we could also consider migrating the Mac and iOS ones to use it to respect the `LoginURL` pref key).